scripts/syschdemd.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+[ "${NIXOS_WSL_DEBUG:-}" == "1" ] && set -x
```

Comment:
```suggestion
[[ -v NIXOS_WSL_DEBUG ]] && set -x
```
testing if it is set to anything maybe makes it easier if people accidentally use true